Makefile

```diff
@@ -161,7 +161,7 @@ operator-image:
 push-operator-image: operator-image
 	docker push ${OPERATOR_IMAGE_PATH}
 
-microk8s-operator-update: operator-image
+microk8s-operator-update: install operator-image
```

Comment:
i think it's always already running install now by default coz  `OPERATOR_IMAGE_BUILD_SRC ?= true
`

Reply:
ah right